10.0/galera/run.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+if [ "$NODE_IP" ]; then
+	echo "wsrep_node_address=$NODE_IP" >> /etc/mysql/conf.d/galera.cnf
+fi
+
+if [ "$MYSQL_CLUSTER_NAME" ]; then
+	echo"wsrep_cluster_name=$MYSQL_CLUSTER_NAME" >> /etc/mysql/conf.d/galera.cnf
+fi
+
+if [ "$CLUSTER_ADDRESS" ]; then
+	export MYSQL_ALLOW_EMPTY_PASSWORD="yes"
+fi
+
+echo "wsrep_cluster_address=${CLUSTER_ADDRESS-gcomm://}" >> /etc/mysql/conf.d/galera.cnf
+
+sed '/mysql_tzinfo_to_sql/d' -i /docker-entrypoint.sh
```

Comment:
Correct. It should be disabled as default for the galera images. As it is slow (5 min or something), and it synced when it join the cluster if present.